LSP.sublime-settings

```diff
@@ -180,6 +180,9 @@
    // or a custom keybinding.
   "show_inlay_hints": false,
 
+  // Truncate limit for inlay hints. Truncates and adds an ellipsis at the end.
+  "inlay_truncate_limit": 1000,
```

Comment:
Call it `inlay_hints_truncate_limit` to make it clear that it relates to the `show_inline_hints` above.

Reply:
Done in [528a430](https://github.com/sublimelsp/LSP/pull/2514/commits/528a43071c4d24d566fc9b251fcd77b05a39a88b)